### PR TITLE
[#IA] Transfering files using http and emailing

### PR DESCRIPTION
**Pull request by maja-tepavcevic**

### CODE REVIEW

File: IntegrationAdapters/SavedList2020-12-07.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (milicasiriski) [inline IntegrationAdapters/SavedList2020-12-07.txt]: If you don't have any more use for these reports you can delete them

File: IntegrationAdapters/Services/MedicineReportService.cs [hunk 2]
```diff
@@ -36,14 +37,39 @@ public List<MedicineReport> GetByDateInterval(TimeInterval timeInterval)
...
+
+        public void SendNotificationAboutReport(String myFile)
+        {
+            try
+            {
+                MailMessage mail = new MailMessage();
+                SmtpClient SmtpServer = new SmtpClient("smtp.gmail.com");
+
+                mail.From = new MailAddress("hospitallhospital@gmail.com");
+                mail.To.Add("ppharmacy98@gmail.com");
```
Comment (milicasiriski) [inline IntegrationAdapters/Services/MedicineReportService.cs]: Extract these emails into constants so you can change them if needed

File: IntegrationAdapters/Services/SftpService.cs [hunk 3]
```diff
@@ -10,9 +10,15 @@ namespace IntegrationAdapters.Services
 {
     public class SftpService : ISftpService
     {
+        public SftpConfig config = new SftpConfig
+        {
+            Host = "192.168.0.23",
+            Port = 22,
+            Username = "tester",
+            Password = "password"
```
Comment (milicasiriski) [inline IntegrationAdapters/Services/SftpService.cs:18]: same here, you can make a class that contains all those constants as static fields and use them

File: IntegrationAdapters/Services/SftpService.cs [hunk 4]
```diff
@@ -54,8 +53,28 @@ public bool SendFile(string fileName)
...
+        {
 
+            using (var client = new SftpClient(config.Host, config.Port, config.Username, config.Password))
+            {
+                client.Connect();
+                if (client.IsConnected)
+                {
+                    using (var fileStream = new FileStream(fileName, FileMode.Create))
+                    {
+                        client.BufferSize = 4 * 1024;
```
Comment (milicasiriski) [inline IntegrationAdapters/Services/SftpService.cs:66]: should this be hard coded?

**Review by milicasiriski — COMMENTED** [review]
Good PR! Have you talked to the product owner about hard coding the email addresses and other parameters? Should that be done differently?

**Review by milicasiriski — APPROVED** [review]

**Review by lukaDoric — APPROVED** [review]

**Pull request merged**